src/styles/utils/variables.scss

```diff
@@ -0,0 +1,8 @@
+$color-main-accent: #060b35;
+$color-secondary: #616070;
+$color-accent-blue: #00acdc;
+$color-hover-button: white;
+$color-hover-title: #34568b;
+$navElementHeight: 23px;
+$fontFamily:"Roboto", "Helvetica", sans-serif;
+$transitionTime: 0.3s;
```

Comment:
Try to maintain a single naming style for variables ( only `kebab-case` or only `camelCase`).
It is also better to place the variable type at the end ( `$main-accent-color`, `$secondary-color` ).

Reply:
okay, i'll change everything to kebab-case

about the color naming, I just used the example from the checklist so that's why its like that :)
BAD example:
```scss
$main: #060b35;
$secondary: #616070;
$blue: #00acdc;
```
GOOD example:
```scss
$color-main-accent: #060b35;
$color-secondary: #616070;
$color-accent-blue: #00acdc;
```